src/main/java/org/server/mbtiliarserver/game/domain/Participant.java

```diff
@@ -0,0 +1,20 @@
+package org.server.mbtiliarserver.game.domain;
+
+public class Participant {
+
+    Long id;
+    String nickName;
```

Comment:
접근 제어자 수정이 필요합니다. 그리고 이름 변경도 필요합니다.

```suggestion
    private Long id;
    private String nickname;
```